doc/source/conduct/code_of_conduct.rst

```diff
@@ -0,0 +1,169 @@
+scikit-image Code of Conduct
+============================
+
+scikit-image adopts the `SciPy code of conduct
+<https://github.com/scipy/scipy/blob/master/doc/source/dev/conduct/code_of_conduct.rst>`__.
```

Comment:
Should this be a single underscore? From http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks : 

```rest
Use  `Link text <https://domain.invalid/>`_ for inline web links.
```

Reply:
Same for other links below.

Reply:
https://necromuralist.github.io/posts/restructured-text-anonymous-links/

Reply:
thanks for the link, it didn't know this either